### PR TITLE
SM model train scripts

### PR DESCRIPTION
**Pull request by artidoro**
Added softmax model train scripts.
Notice that we are using enc_num_layers = 1 in all our experiments so far (that was the number indicated in the paper) but we saw that it was not reflected in code.
Uncomment that line to make the num layer of the encoder 2.

### CODE REVIEW

File: scripts/vmf/train-vmf-enfr.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@ python main.py \
     --device cuda \
     --dropout 0.0 \
     --enc_bidirectional \
+#    --enc_num_layers 2\
```
Comment (brendon-boldt) [inline scripts/vmf/train-vmf-enfr.sh]: This will comment out the rest of the lines in the command, if I am not mistaken, because `\` is a line continuation and `#` is a line comment.

**Review by brendon-boldt — CHANGES_REQUESTED** [review]

File: scripts/vmf/train-vmf-enfr.sh (hunk 1 above)
Comment (artidoro) [reply scripts/vmf/train-vmf-enfr.sh]: Ok so I'll move it to the end of the command.

**Review by brendon-boldt — APPROVED** [review]
